admin_guide/build_defaults_overrides.adoc

```diff
@@ -0,0 +1,78 @@
+= Setting Build Defaults and Overrides
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+The BuildDefaults admission control plugin allows an administrator to set global defaults for settings such as the git
+HTTP and HTTPS proxy, and default envrionment variables. These defaults will not overwrite values that have been configured
+for a particular build. However, if they are not present, they will be set as the default value.
+
+The BuildOverrides admission control plugin allows overriding a setting in a build, regardless of the value stored in the
+build. It currently supports overriding the ForcePull flag on a build strategy to enforce always pulling down of images.
+This ensures that a user can only build with an image that they are allowed to pull.
+
+=== Setting Build Defaults
+
+To configure the BuildDefaults plugin, add a configuration for it in the master-config.yaml file:
+
+====
+
+[source,yaml]
+----
+kubernetesMasterConfig:
+  admissionConfig:
+    pluginConfig:
+      BuildDefaults:
+        configuration:
+          apiVersion: v1
+          kind: BuildDefaultsConfig
+          gitHTTPProxy: http://my.proxy:8080 <1>
+          gitHTTPSproxy: https://my.proxy:8443 <2>
+          env:
+          - Name: HTTP_PROXY <3>
+            Value: http://my.proxy:8080
+          - Name: HTTPS_PROXY <4>
+            Value: https://my.proxy:8443
+          - Name: CUSTOM_VAR <5>
+            Value: custom_value
+----
+
+<1> Sets the HTTP proxy to use when pulling source code from git
+<2> Sets the HTTPS proxy to use when pulling source code from git
+<3> Default environment variable that sets the HTTP proxy to use during the build. This is what is used for pulling
+build artifacts and any other external access during the assemble/build phase.
+<4> Default environment variable that sets the HTTPS proxy to use during the build.
+<5> Arbitrary environment variable that will be added to every build.
```

Comment:
why is this one arbitrary and the others are defaults?  i think you can just say "Additional default environment variable that will be..."